tests/model/test-brep.cpp

```diff
@@ -504,10 +505,21 @@ void set_geometry( geode::BRepBuilder& builder,
         builder.line_mesh_builder( line_uuids[i] )->create_edge( 0, 1 );
     }
 
-    builder.surface_mesh_builder( surface_uuids[0] )->create_point( points[0] );
-    builder.surface_mesh_builder( surface_uuids[0] )->create_point( points[1] );
-    builder.surface_mesh_builder( surface_uuids[0] )->create_point( points[2] );
-    builder.surface_mesh_builder( surface_uuids[0] )
+    builder
+        .surface_mesh_builder< geode::TriangulatedSurface3D >(
+            surface_uuids[0] )
+        ->create_point( points[0] );
```

Comment:
Are theses changes for the test purpose, or mandatory?

Reply:
Only for testing the new template parameter. If you only need a SurfaceMeshBuilder, nothing changes.